frontend/javascripts/oxalis/view/plane_view.ts

```diff
@@ -40,46 +50,63 @@ class PlaneView {
   trigger: (...args: Array<any>) => any;
   // @ts-expect-error ts-migrate(2564) FIXME: Property 'listenTo' has no initializer and is not ... Remove this comment to see the full error message
   listenTo: (...args: Array<any>) => any;
-  cameras: OrthoViewMap<THREE.OrthographicCamera>;
+  cameras: OrthoViewCameraMap;
   throttledPerformIsosurfaceHitTest: (arg0: [number, number]) => THREE.Vector3 | null | undefined;
+  storePropertyUnsubscribers: Array<(...args: Array<any>) => any>;
+  tdViewUseOrthographicCamera: boolean;
 
   running: boolean;
   needsRerender: boolean;
 
   constructor() {
     _.extend(this, BackboneEvents);
-
+    this.storePropertyUnsubscribers = [];
+    this.tdViewUseOrthographicCamera = true;
+    this.bindToEvents();
     this.throttledPerformIsosurfaceHitTest = _.throttle(
       this.performIsosurfaceHitTest,
       ISOSURFACE_HOVER_THROTTLING_DELAY,
     );
     this.running = false;
     const { scene } = getSceneController();
     // Initialize main THREE.js components
-    const cameras: any = {};
+    const partialCameraMap: any = {};
 
-    for (const plane of OrthoViewValues) {
-      // Let's set up cameras
+    const getNewCamera = (name: string, isOrthographic: boolean) => {
       // No need to set any properties, because the cameras controller will deal with that
-      cameras[plane] = new THREE.OrthographicCamera(0, 0, 0, 0);
+      const newCamera = isOrthographic
+        ? new THREE.OrthographicCamera(0, 0, 0, 0)
+        : new THREE.PerspectiveCamera(45, 1, 50, 1000);
       // This name can be used to retrieve the camera from the scene
-      cameras[plane].name = plane;
-      scene.add(cameras[plane]);
+      newCamera.name = name;
+      scene.add(newCamera);
+      return newCamera;
+    };
+
+    for (const plane of OrthoViewValuesWithoutTDView) {
+      // Let's set up cameras
+      partialCameraMap[plane] = getNewCamera(plane, true);
     }
+    partialCameraMap[OrthoViews.TDView] = {
+      [TDCameras.OrthographicCamera]: getNewCamera(`${OrthoViews.TDView}_Orthographic`, true),
+      [TDCameras.PerspectiveCamera]: getNewCamera(`${OrthoViews.TDView}_Perspective`, false),
+    };
+    const cameras: OrthoViewCameraMap = partialCameraMap;
```

Comment:
The workaround `partialCameraMap`is needed to get the cameras into a single object. Other approaches lead to typescript errors.